core/src/main/java/org/elasticsearch/index/SearchSlowLog.java

```diff
@@ -73,20 +73,17 @@
     public static final Setting<TimeValue> INDEX_SEARCH_SLOWLOG_THRESHOLD_FETCH_TRACE_SETTING =
         Setting.timeSetting(INDEX_SEARCH_SLOWLOG_PREFIX + ".threshold.fetch.trace", TimeValue.timeValueNanos(-1),
             TimeValue.timeValueMillis(-1), Property.Dynamic, Property.IndexScope);
-    public static final Setting<Boolean> INDEX_SEARCH_SLOWLOG_REFORMAT =
-        Setting.boolSetting(INDEX_SEARCH_SLOWLOG_PREFIX + ".reformat", true, Property.Dynamic, Property.IndexScope);
     public static final Setting<SlowLogLevel> INDEX_SEARCH_SLOWLOG_LEVEL =
         new Setting<>(INDEX_SEARCH_SLOWLOG_PREFIX + ".level", SlowLogLevel.TRACE.name(), SlowLogLevel::parse, Property.Dynamic,
             Property.IndexScope);
 
+    private static final ToXContent.Params FORMAT_PARAMS = new ToXContent.MapParams(Collections.singletonMap("pretty", "false"));
```

Comment:
good point, I think I went for pretty as default there because it's called by `toString`, and that made more sense in most of the cases, at least for queries and java api builders. Those methods are there solely for logging and debugging at this point. It does make sense to pass in false then in this specific case.